### DISCUSSION TITLE
.NET MAUI 8.0.6 (Service Release 1)

### DISCUSSION THREAD

**Discussion by davidortinau** [dotnet/maui#20559] [Announcements]
This service release is now available in Visual Studio 17.8.6, 17.9, and 17.10 Preview 1 on Windows, and the dotnet CLI across all platforms.

The focus of our service releases is resolving high-impact bugs for developers upgrading their applications from Xamarin to .NET 8. 

* [Release notes](https://github.com/dotnet/maui/releases/tag/8.0.6)

**Comment by DDHSchmidt** [comment]
That's the Service Release from January, isn't it? Why a discussion for that now?
I'm more eager for SR2. I see a [version 8.0.7 on Nuget](https://www.nuget.org/packages/Microsoft.Maui.Controls.Core/8.0.7), but no accompanying Github-release?

Also: [The workload associations seem to have been screwed up](https://github.com/dotnet/maui/issues/20565)?

**Reply by Brosten** [reply]
I guess this is what's included in SR2?
https://github.com/orgs/dotnet/projects/194/views/21

**Reply by gabsamples6** [reply]
When is the SR2 available?  also all the critical fixes on keyaboard -safe area seems to have been moved again to next SR.. 
hmmm - I hope I am wrong.

**Reply by Brosten** [reply]
Isn't this SR2?
https://www.nuget.org/packages/Microsoft.Maui.Core/8.0.7

**Reply by davidortinau** [reply]
Yes it went up to NuGet last night, and release notes are now published.

**Comment by ghost** [comment]
Hello everyone :) I would like to report issues with the integration .NET MAUI project with xUnit. I don't know why but if you want to add the xUnit project into solution target .NET 8, there are a lot of issues. I did the same configuration as for .NET 6 or .NET 7, e.g. like here https://jesseliberty.com/2022/11/21/net-maui-forget-me-not-7-unit-testing/

**Reply by davidortinau** [reply]
I recommend opening a new discussion with a title others can find. @jfversluis might be able to advise on this.

**Reply by ghost** [reply]
Thanks @davidortinau

**Reply by jfversluis** [reply]
@angrynerds-krzysztof please open something separate from this and add details about the issues you are seeing that were not there in .NET 6 & 7. Feel free to tag me or send the link to me otherwise if I don't respond timely. Thanks!
